docs/pages/guides/store.jsx

```diff
@@ -0,0 +1,13 @@
+import { reducer as formReducer } from 'redux-form';
+import { createStore, combineReducers } from 'redux';
+
+const rootReducer = combineReducers({
```

Comment:
Or maybe this file even need to be inside example. So somebody can just copy and paste it to the codesandox